Description: Don't install ChangeLog (already handled by dh_installchangelogs)
Origin: vendor
Forwarded: not-needed
Author: Alessandro Ghedini <ghedo@debian.org>
Reviewed-by: Alessandro Ghedini <ghedo@debian.org>
Last-Update: 2013-08-31

--- a/Makefile.am
+++ b/Makefile.am
@@ -74,4 +74,4 @@
 rpm-ia32: dist
 	CFLAGS="-O2 -m32 -march=i686" rpmbuild --target i686-unknown-linux --clean -ta $(PACKAGE)-$(VERSION).tar.gz
 
-doc_DATA = README TODO NEWS ChangeLog AUTHORS
+doc_DATA = README TODO NEWS AUTHORS
